### PR TITLE
Added Group `file`

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -981,7 +981,161 @@ Outputs the list of peers that were removed.
...
+Interact with IPFS objects representing Unix filesystems.
+
+This provides a familiar interface to file systems represented
+by IPFS objects, which hides IPFS implementation details like layout
+objects (e.g. fanout and chunking).
+
+This command can't be called directly.
+
+## ls [GET /file/ls{?arg}]
+List directory contents for Unix-filesystem object
```
Comment (dignifiedquire) [inline apiary.apib]: Inconsistent spelling of `Unix-filesystem` I suggest to leave out the `-`

File: apiary.apib [hunk 2]
```diff
@@ -981,7 +981,161 @@ Outputs the list of peers that were removed.
...
+
+Retrieves the object named by <ipfs-or-ipns-path> and displays the
+contents.
+
+The JSON output contains size information.  For files, the child size
```
Comment (dignifiedquire) [inline apiary.apib]: double blank space

File: apiary.apib [hunk 3]
```diff
@@ -981,7 +981,161 @@ Outputs the list of peers that were removed.
...
+is the total size of the file contents.  For directories, the child
```
Comment (dignifiedquire) [inline apiary.apib]: double blank space

**Comment by RichardLitt** [comment]
@dignifiedquire Fixed all comments. RFM?

**Comment by dignifiedquire** [comment]
LGTM

**Comment by RichardLitt** [comment]
Thanks, @dignifiedquire!

**Pull request merged**